README.md

```diff
@@ -1,20 +1,27 @@
-# Multiomic ALS signatures highlight sex differences and molecular subclusters and identify the MAPK pathway as therapeutic target
```

Comment:
That is a very strange title.  Shouldn't that be the title of the project and not some strange abbreviation?

Reply:
That is the tile you chose, I did not change it